src/utils.js

```diff
@@ -12,3 +12,7 @@ export function parseLength(obj, acceptStr) {
   }
   return null;
 }
+
+export function isValidAxis(axis) {
```

Comment:
This could be put into the ScrollTimeline class as you can call this.method inside the constructor.  If we want to make it a static, then we can add it as a method to ScrollTimeline's prototype. This approach might be even better.

```
ScrollTimeline.isValidAxis(axis);
```

This results in less exports to deal with and provides better encapsulation.

Reply:
.done